go.mod

```diff
@@ -1,6 +1,6 @@
 module open-cluster-management.io/policy-generator-plugin
 
-go 1.21
```

Comment:
1.22?

Reply:
@dhaiducek  Is this ok?

Reply:
Yes--it probably doesn't matter for the generator, but my script used the same version per ACM version across the board for consistency. For ACM 2.10 and below, we hadn't bumped the `controller-gen` version since we needed to be able generate `v1beta1` CRDs. But the older version of `controller-gen` isn't compatible with Go 1.22, so this needs to stay at 1.21.